app/src/main/java/com/example/parkingsystem/mvp/presenter/ParkingSpaceReservationPresenter.java

```diff
@@ -20,6 +20,15 @@ public void onButtonParkingSpaceReservationPickerPressed(DatePickerDialog.OnDate
         view.showDatePickerDialog(listener);
     }
 
+    @Override
+    public void onButtonParkingSpaceReservationSavePressed() {
```

Comment:
please unify the 3 model methods and both vie methods

Reply:
Done!